scripts/aws/derive_ses_smtp_password.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+# Copy-paste-edited from https://docs.aws.amazon.com/ses/latest/DeveloperGuide/smtp-credentials.html
+# Example Usage:
+#   CREDENTIAL_FILE=/path/to/downloaded/iam_user_name_accessKeys.csv
+#   SES_SECRET=$(cat ${CREDENTIAL_FILE} | head -n2 | tail -n1 | cut -d',' -f2)
+#   ./scripts/aws/derive_ses_smtp_password.py --region us-east-1 --secret ${SES_SECRET}
+
+import hmac
+import hashlib
+import base64
+import argparse
+
+# Values that are required to calculate the signature. These values should
+# never change.
+DATE = "11111111"
```

Comment:
This is my birthday!

Reply:
Wow you're turning 909 years old in just a few months!